contributors/design-proposals/node/pod-resource-management.md

```diff
@@ -16,7 +16,7 @@ to Kubernetes. It outlines the implementation and associated rollout plan.
 ## Introduction
 
 Kubernetes supports container level isolation by allowing users
-to specify [compute resource requirements](resources.md) via requests and
+to specify [compute resource requirements](../scheduling/resources.md) via requests and
```

Comment:
This breaks when moving the file. Why not use: `/contributors/design-proposals/scheduling/resources.md`?

Reply:
You are right. I will change it as you said.

Reply:
Great thanks.
Also curious sidenote:
Same repo links (as we have here) can just use the relative path.
Cross repo links can be done using something like: "git.k8s.io/community/contributors/design-proposals/scheduling/resources.md"
And issue links can be done using something like: "http://issue.k8s.io/1"